api/app/signals/apps/services/domain/filescanner.py

```diff
@@ -0,0 +1,44 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2021 Gemeente Amsterdam
+"""
+Check file types, both for fresh uploads and stored files.
+"""
+import mimetypes
+
+import magic
+
+ALLOWED_MIME_TYPES = ['image/jpeg', 'image/png', 'image/gif']
```

Comment:
Let's make this a setting and the `['image/jpeg', 'image/png', 'image/gif']` could be the default. This way we can easily extend the allowed file types.